aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -149,12 +149,17 @@ module aptos_framework::genesis {
             // Transfer coins from the root account to the validator, so they can stake and have non-zero voting power
             // and can complete consensus on the genesis block.
             coins::register<AptosCoin>(&owner_account);
-            aptos_coin::mint(&aptos_framework_account, *owner, amount);
+            aptos_coin::mint(aptos_framework, *owner, amount);
             stake::add_stake(&owner_account, amount);
             stake::join_validator_set_internal(&owner_account, *owner);
 
             i = i + 1;
         };
+
+        // Destroy the aptos framework account's ability to mint coins now that we're done with setting up the initial
+        // validators.
+        aptos_coin::destroy_mint_cap(aptos_framework);
+
```

Comment:
we shouldn't do this, we should make this either opt-in or opt-out -- that way testnets can continue to have unlimited faucets.

Reply:
The faucet in tests/testnets use the core resources account, which already has mint capability and can be controlled via the is_test flag in genesis configurations

Reply:
He added the `is_test` flag to allow this to be controlled.

Though why is it destroyed here?  I don't see the configuration for `is_test`

Reply:
We need this removal for both tests/testnets and mainnet. For testnets, the core resources account already has the mint capability and will keep it. For both testnets and mainnet, aptos framework account only needs temporary mint power to mint to initial validators and accounts. Once it's done, it no longer needs mint cap.